README.md

```diff
@@ -11,12 +11,13 @@
 
 Hint: click on the name to be taken to the appropriate wrapper code
 
-* [Python](wrappers/python)
```

Comment:
Can you please revert this change and add Mathcad somewhere in the list?  They are roughly sorted in terms of importance.

Reply:
No problem.  I just sorted them alphabetically and put Mathcad in the appropriate position.  I'll pull that piece out.  Wasn't aware of an "importance" ranking.  Where should Mathcad go?

Reply:
Done.